besu/src/main/java/org/hyperledger/besu/controller/QbftBesuControllerBuilder.java

```diff
@@ -86,7 +88,8 @@
   private BftEventQueue bftEventQueue;
   private BftConfigOptions bftConfig;
   private ValidatorPeers peers;
-  private final BlockInterface blockInterface = new BftBlockInterface();
+  private final BftExtraDataEncoder bftExtraDataEncoder = new IbftExtraDataEncoder();
```

Comment:
will  `new IbftExtraDataEncoder();` become `QbftExtraDataEncoder` in future PR?

Reply:
Going to change it now since I already have a QbftExtraDataEncoder which is a copy of IbftExtraDataEncoder.